src/internal/util/subscribeTo.ts

```diff
@@ -17,7 +17,7 @@ export const subscribeTo = <T>(result: ObservableInput<T>): (subscriber: Subscri
   } else if (isArrayLike(result)) {
     return subscribeToArray(result);
   } else if (isPromise(result)) {
-    return subscribeToPromise(result as Promise<any>);
+    return subscribeToPromise(result);
```

Comment:
This change is fine, though. The type assertion in the current codebase is unnecessary and can be removed.